src/config.js

```diff
@@ -17,6 +17,35 @@ type ApplyConfigToArgvParams = {|
   configFileName: string,
 |};
 
+type ApplySubOptionsToArgvParams = {|
+  commandExecuted: string,
+  argv: Object,
+  mapCommandToSubOpts: Object,
+  configObject: Object,
+  defaultValues: Object,
+  configFileName: string,
+|};
+
+export function applySubOptionsToArgv({
+  commandExecuted,
+  argv,
+  mapCommandToSubOpts,
+  configObject,
+  defaultValues,
+  configFileName,
+}: ApplySubOptionsToArgvParams): Object {
+  // cannot use spread operator because Flow complains. See: https://github.com/facebook/flow/pull/3381/
+  const newArgv = JSON.parse(JSON.stringify(argv));
```

Comment:
actually, there's another problem. Read the Flow error more closely, it explains exactly what's happening (be sure you are looking at the right line numbers).